ZalikovkaC#/Program.cs

```diff
@@ -10,26 +10,26 @@ class Program
     {
         static void Main(string[] args)
         {
-            int k_1 = 6, k_2 = 5, k_3 = 4, k_4 = 4, k_5 = 4, k_6 = 3, k_7 = 2, k_8 = 2;
-            int VM, FZ, INF, HIST, TCA, ECO, ENG, ENG_1, FIZ;
+            const int k_1 = 6, k_2 = 5, k_3 = 4, k_4 = 4, k_5 = 4, k_6 = 3, k_7 = 2, k_8 = 2;
```

Comment:
What does k_1 ... k_8 mean? Can you link these koefs to subjects? 
Names must be meaningful.

Reply:
Think about saving this constants into separate file.